### DISCUSSION TITLE
Make Token invalid in keycloak

### DISCUSSION THREAD

**Discussion by zoroglu** [keycloak/keycloak#21900] [Q&A]
There is no direct api service where I can invalidate the Token before the token I got from Keycloak expires. how do i make the token invalid in Keycloak?

**Comment by danielFesenmeyer** [comment]
I think logging out the session associated with the user should work.
There is an endpoint at the user which logs out ALL sessions of a user: https://www.keycloak.org/docs-api/22.0.1/rest-api/#:~:text=POST%20/admin/realms/%7Brealm%7D/users/%7Bid%7D/logout

But this of course does not only logout a specific session.

Maybe someone knows a different possibility?

**Comment by zoroglu** [comment]
When I use this service, I can still access services with tokens.

**Reply by danielFesenmeyer** [reply]
Can you still use it with Keycloak itself, or only with thirdparty-services?
